docs/resim/metrics/batch_metrics.md

```diff
@@ -1,5 +1,57 @@
 # Batch metrics
 
-Our batch metrics system is currently functional but in an early phase where the interface is not fully finalized - as a result, the docs are a WIP. 
+Batch metrics are very similar to job metrics, but are computed after all the job metrics are computed, using the job-level metrics and metrics data as the "input."
 
-**If you are a customer and have any questions, feel free to reach out!**
+> Example Batch Metric 1: a weighted average of a job-level scalar metric, across all the jobs present.
+> Example Batch Metric 2: a histogram of a single series that is computed by all metrics jobs, after merging these series into one by appending.
+
+Batch metrics can also be used to gate whether batches pass or fail, though this feature is currently still a WIP.
+
+## Batch metrics mode
+
+As mentioned in [the Metrics builds docs](./metrics_builds.md), batch metrics use the same single image as job metrics. 
+
+This image can differentiate whether it's computing in `job mode` or `batch mode`, based off whether the file `/tmp/resim/inputs/batch_metrics_config.json` is present. If it is present, it should be computing batch metrics.
+
+## Computing batch metrics
+
+The batch metrics config is a simple json with three fields: an auth token, an API URL, and a batch ID.
```

Comment:
[nit] Maybe make it more explicit that the worker provides this so customers don't think they need to do anything special to get the token?
```suggestion
The batch metrics config provided to the batch metrics runner on launch is a simple json with three fields: an auth token, an API URL, and a batch ID.
```

Reply:
Looks good!